service-01/meta.json

```diff
@@ -14,18 +14,11 @@
             "Request.asn1",
             "ErrorCodes.asn1"
           ],
-          "imports": [
+          "requires": [
             {
-              "from": "PacketTypes",
-              "types":[
-                "PacketVersionNumberValue",
-                "PacketSequenceControl",
-                "Packet-ID"
-              ]
-            },
-            {
-              "from": "TC-Packet",
-              "types": ["TC-PacketType"]
+	            "module": "CCSDS",
```

Comment:
indent? (spaces vs. tabs?)

Reply:
Spaces. I was probably using other editor for quick typo fix